frontend/src/scenes/billing/CurrentUsage.tsx

```diff
@@ -0,0 +1,59 @@
+import { Card, Progress, Tooltip } from 'antd'
+import { useValues } from 'kea'
+import React from 'react'
+import { userLogic } from 'scenes/userLogic'
+import { billingLogic } from './billingLogic'
+
+export function CurrentUsage(): JSX.Element {
+    const { eventAllocation, percentage, strokeColor } = useValues(billingLogic)
+    const { user } = useValues(userLogic)
+    const plan = user?.billing?.plan
+
+    return (
+        <>
+            <div className="space-top" />
+            <Card title="Current monthly usage">
+                {user?.billing?.current_usage && (
+                    <>
+                        Your organization has used{' '}
+                        <Tooltip title={`${user.billing.current_usage.value.toLocaleString()} events`}>
+                            <b>{user.billing.current_usage.formatted}</b>
+                        </Tooltip>{' '}
+                        events this month.{' '}
+                        {eventAllocation?.value && (
+                            <>
+                                You can use up to <b>{eventAllocation.formatted}</b> events per month.
```

Comment:
Core to PostHog/posthog-production#69 rather than this PR, but I'd really really like to avoid formatting server-side going forward unless it's absolutely necessary.

Reply:
For context, being addressed in #3360